Networking/Networking/Remote/MediaRemote.swift

```diff
@@ -85,13 +85,18 @@ public class MediaRemote: Remote, MediaRemoteProtocol {
         ]
 
         let path = "sites/\(siteID)/media"
-        let request = DotcomRequest(wordpressApiVersion: .wpMark2,
-                                    method: .get,
-                                    path: path,
-                                    parameters: parameters)
-        let mapper = WordPressMediaListMapper()
+        do {
+            let request = try DotcomRequest(wordpressApiVersion: .wpMark2,
+                                        method: .get,
+                                        path: path,
+                                        parameters: parameters,
+                                        availableAsRESTRequest: true)
```

Comment:
super nit: the indents need to be updated here 😅

Reply:
Done in 5144ba9af8c94c0a255939012259a95883c39c38